pep-0695.rst

```diff
@@ -809,9 +819,13 @@ It also adds an AST node type that represents a type parameter.
         | ParamSpec(identifier name)
         | TypeVarTuple(identifier name)
 
+Bounds and constraints are represented identically in the AST. In the implementation,
+any expression that is syntactically a tuple is treated as a constraint, and any other
+expression is treated as a bound.
```

Comment:
Isn't this a little too pragmatic and subtle?

Reply:
Possibly. An alternative could be to recognize this in the parser and produce a separate AST node. That would be more complicated to implement, and possibly create an ambiguous parse tree.